### PR TITLE
Fixed styles override issue

### LINKED ISSUE #108: Existing components are affected
- [ ] `h1` tag has extra margin
- [ ] Nested neetoUI Dropdown not working

Issue - [video](https://www.loom.com/share/ff3a87fa0ab84ec58c5907d4e425e3a1)

### PR DESCRIPTION
**Pull request by goutham-subramanyam**
Fixes #108 

- Removed h1 overrides from normalize
- Updated dropdown classname to prevent conflict with neetoUI

Note: We may have to update the classnames for other common components as well if we face issues like this again.

@amaldinesh7 _a Please review.

### CODE REVIEW

**Comment by amaldinesh7** [comment]
@goutham-subramanyam If there is a chance of styles getting overridden, can't we change all the class names similar to neetoUI?

File: lib/components/Common/Dropdown/index.js [hunk 1]
```diff
@@ -94,8 +94,8 @@ const Dropdown = forwardRef(
     return (
       <div
         ref={wrapperRef}
-        className={classnames("neeto-ui-dropdown__wrapper", {
-          "neeto-ui-dropdown__wrapper--auto-width": autoWidth,
+        className={classnames("nui-dropdown__wrapper", {
```
Comment (amaldinesh7) [inline lib/components/Common/Dropdown/index.js:97]: Thoughts about using prefix `ne` instead of `nui`?

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@goutham-subramanyam _a I've added one comment. Please check.

File: lib/components/Common/Dropdown/index.js (hunk 1 above)
Comment (goutham-subramanyam) [reply lib/components/Common/Dropdown/index.js:97]: Sure

**Comment by goutham-subramanyam** [comment]
@amaldinesh7 _a

**Comment by amaldinesh7** [comment]
LGTM!

**Review by amaldinesh7 — APPROVED** [review]

**Pull request merged**